psi4/driver/procrouting/empirical_dispersion.py

```diff
@@ -175,45 +178,68 @@ def print_out(self, level=1):
 
         core.print_out(self.citation + "\n\n")
 
-        core.print_out("        S6 = %14.6E\n" % self.dash_params["s6"]);
+        if (self.disp_type == 'nl'):
+            return core.print_out("\n")
+
+        core.print_out("        S6 = %14.6E\n" % self.dash_params["s6"])
         if "s8" in self.dash_params.keys():
-            core.print_out("        S8 = %14.6E\n" % self.dash_params["s8"]);
+            core.print_out("        S8 = %14.6E\n" % self.dash_params["s8"])
 
         for k, v in self.dash_params.items():
             if k in ["s6", "s8"]: continue
 
-            core.print_out("    %6s = %14.6E\n" % (k.upper(), v));
+            core.print_out("    %6s = %14.6E\n" % (k.upper(), v))
 
         # Psi auto sets this with no change of deviation
         if (self.disp_type == 'p4') and (self.dtype == "-D2"):
             core.print_out("    %6s = %14.6E\n" % ("A6", 20.0))
 
-        core.print_out("\n");
+        core.print_out("\n")
 
     def compute_energy(self, molecule):
+        if self.disp_type == 'nl':
```

Comment:
this looks even to me poorly solved. Just needed it to stop giving errors to get a working version.